microprofile/WFLY-17137-microprofile-jakarta-ee-10.adoc

```diff
@@ -0,0 +1,185 @@
+= MicroProfile aligned with Jakarta EE 10
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+This issue is about upgrading to most parts of MicroProfile Platform 6, which is aligned with Jakarta EE 10. 'most' is used since a decision has been made to not include the new MicroProfile Metrics version, as the inclusion of Micrometer tracked by https://issues.redhat.com/browse/EAP7-1686[EAP7-1686] provides our observability strategy.
```

Comment:
Please use a WFLY JIRA here.

Micrometer is only in WildFly Preview right now so @jasondlee can tell you the JIRA that tracks inclusion in standard WildFly.

Reply:
THe JIRA to move Micrometer to standard WF is here https://issues.redhat.com/browse/WFLY-17144

Reply:
@kabir ^^^